### PR TITLE
PDP-1322: incorporate comments from meeting

### PR DESCRIPTION
**Pull request by nashtapu-tibco**

### CODE REVIEW

File: dp-cluster-workshop/eks/README.md [hunk 1]
```diff
@@ -711,37 +733,45 @@ config:
...
-| Grafana endpoint  | https://grafana.\<base FQDN\> | this comes from Grafana service                                        |
+| VPC_CIDR             | 10.200.0.0/16                                                                    | from eks recipe                                         |
+| Ingress class name   | nginx                                                                            | used for BWCE                                                     |
+| EFS storage class    | efs-sc                                                                           | used for BWCE EFS storage                                         |
+| EBS storage class    | ebs-gp3                                                                          | used for EMS messaging                                            |
+| BW FQDN              | bwce.\<BASE_FQDN\>                                                               | capability fqdn |
+| Elastic User app logs index   | user-app-1                                                                       | dp-config-es index template                               |
```
Comment (akusingh-tibco) [inline dp-cluster-workshop/eks/README.md:761]: @nashtapu-tibco for the indexes, should we make a note that enter the values which are configured in o11y-data-plane-configuration in CP UI. (user-app-*)

File: dp-cluster-workshop/eks/README.md [hunk 2]
```diff
@@ -711,37 +733,45 @@ config:
...
+| Elastic Search logs index     | service-1                                                                        | dp-config-es index template                               |
+| Elastic Search internal endpoint | https://dp-config-es-es-http.elastic-system.svc.cluster.local:9200               | Elastic Search service                                                |
+| Elastic Search public endpoint   | https://elastic.\<BASE_FQDN\>                                                    | Elastic Search ingress host                                                |
+| Elastic Search password          | xxx                                                                              | Elastic Search password in dp-config-es-es-elastic-user secret                                             |
+| Tracing server host  | https://dp-config-es-es-http.elastic-system.svc.cluster.local:9200               | Elastic Search internal endpoint                                         |
+| Prometheus endpoint  | http://kube-prometheus-stack-prometheus.prometheus-system.svc.cluster.local:9090 | Prometheus service                                        |
```
Comment (akusingh-tibco) [inline dp-cluster-workshop/eks/README.md:767]: @nashtapu-tibco should we change it to this https://prometheus-internal/.<base FQDN> ?

`AVXD0230A:oneimagek8sazclihelmbash akusingh$ kubectl get ingress -A | grep .pro
elastic-system      dp-config-es-apm                    nginx                             apm.dp1.azure.dataplanes.pro                   10.0.111.142     80        43h
elastic-system      dp-config-es-elastic                nginx                             elastic.dp1.azure.dataplanes.pro               10.0.111.142     80        43h
elastic-system      dp-config-es-kibana                 nginx                             kibana.dp1.azure.dataplanes.pro                10.0.111.142     80        43h
ingress-system      nginx                               azure-application-gateway         *.dp1.azure.dataplanes.pro                     20.242.221.106   80, 443   2d20h
ns                  bw-cl35bbtia14iuic85u9g             nginx                             bwce.dp1.azure.dataplanes.pro                  10.0.111.142     80        2d20h
ns                  bwprovisioner-public-nginx          nginx                             bwce.dp1.azure.dataplanes.pro                  10.0.111.142     80        2d20h
ns                  haproxy-artifactmanager             haproxy-dp-cl354fh3nmo40c59br40   *                                                               80        2d20h
ns                  haproxy-bwprovisioner               haproxy-dp-cl354fh3nmo40c59br40   *                                                               80        2d20h
ns                  haproxy-jaeger                      haproxy-dp-cl354fh3nmo40c59br40   *                                                               80        2d20h
ns                  haproxy-o11y-service                haproxy-dp-cl354fh3nmo40c59br40   *                                                               80        2d20h
ns                  oauth2-proxy-cl3571h3nmo40c59br80   nginx                             bwce.dp1.azure.dataplanes.pro                  10.0.111.142     80        2d20h
ns                  provisioner-agent                   haproxy-dp-cl354fh3nmo40c59br40   *                                                               80        2d20h
ns                  tp-dp-monitor-agent                 haproxy-dp-cl354fh3nmo40c59br40   *                                                               80        2d20h
prometheus-system   kube-prometheus-stack-grafana       nginx                             grafana.dp1.azure.dataplanes.pro               10.0.111.142     80        43h
prometheus-system   kube-prometheus-stack-prometheus    nginx                             prometheus-internal.dp1.azure.dataplanes.pro   10.0.111.142     80        43h`

File: dp-cluster-workshop/eks/README.md [hunk 3]
```diff
@@ -711,37 +733,45 @@ config:
...
+| Elastic User app logs index   | user-app-1                                                                       | dp-config-es index template                               |
+| Elastic Search logs index     | service-1                                                                        | dp-config-es index template                               |
+| Elastic Search internal endpoint | https://dp-config-es-es-http.elastic-system.svc.cluster.local:9200               | Elastic Search service                                                |
+| Elastic Search public endpoint   | https://elastic.\<BASE_FQDN\>                                                    | Elastic Search ingress host                                                |
+| Elastic Search password          | xxx                                                                              | Elastic Search password in dp-config-es-es-elastic-user secret                                             |
+| Tracing server host  | https://dp-config-es-es-http.elastic-system.svc.cluster.local:9200               | Elastic Search internal endpoint                                         |
```
Comment (akusingh-tibco) [inline dp-cluster-workshop/eks/README.md:766]: @nashtapu-tibco should we change this to https://grafana.\<BASE_FQDN\>

**Review by syan-tibco — APPROVED** [review]

File: dp-cluster-workshop/eks/README.md (hunk 2 above)
Comment (syan-tibco) [reply dp-cluster-workshop/eks/README.md]: we want to use internal address for DP to use. External one is more for user to use

Comment (syan-tibco) [reply dp-cluster-workshop/eks/README.md]: adding both is also good

**Review by akusingh-tibco — APPROVED** [review]

File: dp-cluster-workshop/eks/README.md (hunk 2 above)
Comment (nashtapu-tibco) [reply dp-cluster-workshop/eks/README.md]: Both are added.

**Pull request merged**